index.js

```diff
@@ -186,8 +186,18 @@ exports.viewAccount = async function (options) {
     if (state && state.amount) {
         state['formattedAmount'] = utils.format.formatNearAmount(state.amount);
     }
-    console.log(`Account ${options.accountId}`);
-    console.log(inspectResponse.formatResponse(state));
+    const storageInM=state.storage_usage/1e+6;
+    const accountDetails={
+        'Account:':`${options.accountId} at block ${state.block_height} (https://explorer.testnet.near.org/blocks/${state.block_hash})`,
+        [chalk.bold('amount:')]:`${state.amount} ${chalk.bold('yn')}`,
```

Comment:
Let's have an `amount` field that would be "smart". If we have 123 yN in our acc - let it be:
Amount:  123 yN

and if it has 100624871403184387199999986 yN, let it be:

Amount: ~1100.6 N

You may use existing `utils` from `near-api-js` and `near-cli` for it or add your own util functions.

At the same time, we need to have a `Raw amount` field,  that can be useful during the development. It will always show the amount in yN. Something like:

**Raw amount**: 100624871403184387199999986 yN